soh/src/overlays/actors/ovl_En_Heishi1/z_en_heishi1.c

```diff
@@ -112,14 +112,20 @@ void EnHeishi1_Init(Actor* thisx, GlobalContext* globalCtx) {
         }
     }
 
+    // eventChkInf[4] & 1 = Got Zelda's Letter
+    // eventChkInf[5] & 0x200 = Got item from impa
+    // eventChkInf[8] & 1 = Ocarina thrown in moat
+    bool rando = gSaveContext.n64ddFlag;
```

Comment:
having this as a variable here isn't a good idea imo, if we want to move away from having n64ddflag used we should clean it up everywhere instead

Reply:
That's fair, I'll change it back. That said, it might be an idea to make a helper function called Randomizer_IsActive() in the future that replaces all the n64dd flags to improve readability and make the n64dd flag less obscure. Not for this PR obviously but yeah.